tests/unittest/_torch/modeling/test_kimi_kda_fp8_packed_prefill.py

```
@@ -0,0 +1,203 @@
+@torch.no_grad()
+def test_fp8_packed_qkv_projection_matches_separate_views() -> None:
+    torch.manual_seed(0)
+    runtime = _make_runtime()
+    mixer = runtime.mixer
+    hidden = torch.randn(1, 193, _Cfg.hidden_size, device="cuda", dtype=torch.bfloat16) * 0.05
+
+    packed = mixer.qkvg_proj(hidden)[..., : 3 * runtime.proj_size]
+    actual = packed.split(runtime.proj_size, dim=-1)
+    expected = (mixer.q_proj(hidden), mixer.k_proj(hidden), mixer.v_proj(hidden))
+
+    for packed_part, separate_part in zip(actual, expected):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)